aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/exceptions/handlers/SelectiveExceptionHandler.java

```diff
@@ -47,6 +49,26 @@ public void handle(String message, Exception cause) {
 
     @Override
     public boolean haltsOn(Class<? extends Exception> exceptionType) {
-        return criticalExceptions.stream().anyMatch(exName -> exceptionType.getName().equalsIgnoreCase(exName));
+        Class<?> managedClass;
+        for (String exName : criticalExceptions){
+            if (exName.startsWith("!")) {
```

Comment:
Please declare a constant fot this.
Then, I'd rather pulled this check out of the current `if... else` block. Logic is as follows: 
1) we test for the current substring being "*". 
2) If not, we've got either class name, or a package name. Leave alone "!" and compare the target exception to this class name or package name
3) If "!" is present, invert the comparison result
4) return if true